airflow/providers/fab/auth_manager/cli_commands/user_command.py

```diff
@@ -106,6 +106,27 @@ def _find_user(args):
     return user
 
 
+@cli_utils.action_cli
+@providers_configuration_loaded
+def user_reset_password(args):
+    """Reset user password user from DB."""
+    user = _find_user(args)
+    if args.use_random_password:
+        password = "".join(random.choices(string.printable, k=16))
+    elif args.password:
+        password = args.password
+    else:
+        password = getpass.getpass("Password:")
+        password_confirmation = getpass.getpass("Repeat for confirmation:")
+        if password != password_confirmation:
+            raise SystemExit("Passwords did not match")
```

Comment:
To prevent code duplication and redundancy, can you put these lines into a utility function and use it in here and `users_create(args)`?

Reply:
done